Backend/cardGroupings/tests/test_Card.py

```diff
@@ -0,0 +1,45 @@
+# tests/test_card.py
+import pytest
+from Card import CardType, Card
+
```

Comment:
Django has testing library that is built upon pytest that allows one to test the entire Django suite e.g. (Models, Views, Authentication, Channels). Furthermore, you can create a base class that inherits the unit.TestCase class that allows one to have a setup function and tear down function for setting up the test cases.